client/components/PubShareDialog/PubShareDialog.js

```diff
@@ -46,7 +46,7 @@ const AccessHashOptions = (props) => {
 	const { viewHash, editHash } = pubData;
 
 	const renderHashRow = (label, hash) => {
-		const url = pubUrl(communityData, pubData, { accessHash: hash });
+		const url = pubUrl(communityData, pubData, { accessHash: hash, isDraft: true });
```

Comment:
It would be better to say `isDraft: !pubData.isRelease` here. While the sharing dialog is effectively only shown on drafts right now, the causal relationship is a little indirect, and is down to the fact that `membersData` isn't computed when showing releases. This dialog should be robust to potential changes here.

Reply:
Agreed. Changed to set `isDraft: !isRelease`